cli/azd/pkg/tools/azcli/apim.go

```diff
@@ -0,0 +1,85 @@
+package azcli
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/apimanagement/armapimanagement"
+)
+
+type AzCliAPIM struct {
```

Comment:
Check if we should call it `AzCliApim` instead.  
Just to keep it the same way as we name other tools' structs